auth_session_timeout/i18n/de.po

```diff
@@ -1,23 +0,0 @@
-# Translation of Odoo Server.
```

Comment:
Thanks for the explanation @pedrobaeza - I think that the issue that you linked is the one that made me think this way. I'll add back 😄